src/cpu/aarch64/cpu_reducer.cpp

```diff
@@ -0,0 +1,597 @@
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
```

Comment:
nit: [here and elsewhere] year change in copyright 2020 ---> 2021

Reply:
OK, I'll fix it.